### PR TITLE
Fixed failing system test 

### LINKED ISSUE #20620: MuonFFTTest fails
The MuonFFTTEst system test seems to be failing on master: 

http://builds.mantidproject.org/view/Master%20Pipeline/job/master_systemtests-win7/467/testReport/SystemTests/MuonFFTTest/MuonFFTTest/

### PR DESCRIPTION
**Pull request by AnthonyLim23**
The FFTTest was failing when using PhaseQuad. This error came from one of the arrays using point data and the other bin edges. They both now use point data. 

**To test:**
Check that the system test MuonFFTTest and unit test for phaseQuad pass. 

Fixes #20620 

**Release Notes** 

Does not need to be in release notes
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by martyngigg — CHANGES_REQUESTED** [review]

File: Framework/Algorithms/src/PhaseQuadMuon.cpp [hunk 1]
```diff
@@ -226,15 +226,14 @@ PhaseQuadMuon::squash(const API::MatrixWorkspace_sptr &ws,
...
-  const double X0 = ws->x(0).front();
+  const double X0 = xPointData.front();
 
   // calculate exponential decay outside of the loop
-  std::vector<double> expDecay(ws->histogram(0).size()); // make a copy
-  {                                                      // limit scope
-    const auto &X = ws->readX(0);
-    for (size_t i = 0; i < X.size(); ++i)
-      expDecay[i] = exp(-(X[i] - X0) / muLife);
+  std::vector<double> expDecay(xPointData.size()); // make a copy
```
Comment (martyngigg) [inline Framework/Algorithms/src/PhaseQuadMuon.cpp]: While looking at this again I've realised we could do this in an even cleaner way:
```c++
std::vector<double> expDecay = xPointData;
std::transform(expDecay.begin(), expDecay.end(), expDecay.begin(), [X0,muLife](double x){ return exp(-(x - X0) / muLife) });
```

What do you think?

Comment (AnthonyLim23) [reply Framework/Algorithms/src/PhaseQuadMuon.cpp]: that looks good to me. I will change it in a mo

**Review by martyngigg — APPROVED** [review]

**Comment by martyngigg** [comment]
@mantidproject/gatekeepers Can we get this merged? It fixes the system tests on Windows.

**Comment by ianbush** [comment]
Just to check, is the release branch being periodically merged back into master?

**Comment by martyngigg** [comment]
@ianbush Yep, usually once a day.

**Pull request merged**